server/src/main/java/com/linecorp/centraldogma/server/internal/storage/repository/git/GitRepository.java

```diff
@@ -419,7 +419,7 @@ private Map<String, Entry<?>> blockingFind(
                 } else {
                     switch (entryType) {
                         case JSON:
-                            entry = Entry.ofJson(normRevision, path, Jackson.nullNode);
+                            entry = Entry.ofJson(normRevision, path, "");
```

Comment:
Understood that with this change, `Entry`s that are of JSON type generated by the server are guaranteed to always have non-null `rawContent`.